asst2/CODE/datastructs.c

```diff
@@ -157,14 +158,14 @@ void free_map(hashmap* h) {
     while (node) {
       hash_node* next = node->next;
       // context, linkedlists free'd earlier
-      free(node->value); //free tcb
-      free(node);
+      mydeallocate(node->value, __FILE__, __LINE__, LIBRARYREQ); //free tcb
+      free(node); //calloc'd
       node = next;
     }
   }
   
   free(h->buckets);
```

Comment:
is this an oops?

Reply:
nah, two different values being free'd, one uses calloc and one uses myallocate

Reply:
I don't think we can use calloc either